tests/visual/scenes/filters/alpha-filter.scene.ts

```diff
@@ -0,0 +1,29 @@
+import { AlphaFilter } from '../../../../src/filters/defaults/alpha/AlphaFilter';
+import { Container } from '../../../../src/scene/container/Container';
+import { Graphics } from '../../../../src/scene/graphics/shared/Graphics';
+
+import type { TestScene } from '../../types';
+
+export const scene: TestScene = {
```

Comment:
Should we also have a png snapshot accompanying the test?

Reply:
yup, uploaded 👍